server/src/lib/model/db/import-locales.ts

```diff
@@ -258,6 +258,136 @@ const VARIANTS: Variant[] = [
     variant_name: 'Адыгэбзэ (Къэбэрдей, Кирил, Сирие)',
     variant_token: 'kbd-Cyrl-SY',
   },
+  {
+    locale_name: 'pwn',
+    variant_name: 'pinayuanan a tjinuvecekadanan (中排灣語)',
```

Comment:
Capitalisation, should be *Pinayuanan a Tjinuvecekadanan* or *Pinayuanan a tjinuvecekadanan* ?